BEGINNER_GUIDE.md

```diff
@@ -0,0 +1,167 @@
+## Beginner's Guide
+
+First of all, we welcome you to the open-source contribution.
+
+This guide will help you to make your first contribution to the AnitaB.org.
+
+Read the project's README.md file and get a basic idea of what the project is about.
+
+Read the [Contributing guidelines](https://github.com/anitab-org/vms/blob/develop/CONTRIBUTING.md) and [Reporting guidelines](http://systers.io/reporting-guidelines). These guidelines need to be kept in mind while contributing to any project.
+---
+
+### Getting your first issue assigned
+Now since you have chosen your project, follow the steps mentioned below to get your first issue assigned.
+
+**Step 1:** Go to the **Issues**  tab of the selected project repository.
+
+![issue_tab](https://user-images.githubusercontent.com/64194946/93016427-b36c8580-f5de-11ea-82ee-57111edad230.png)
+
+
+**Step 2:** Now, from the list of the issues, search for the issue that you are comfortable working upon. Make sure that this issue is not assigned to someone else.
+
+For example, issue #1114 and #1112 don't have any assignee whereas issue #1111 has one.
+
+![issue_list](https://user-images.githubusercontent.com/64194946/93016491-46a5bb00-f5df-11ea-964a-5f8ff0c940a1.png)
+
+
+**Step 3:** Click on the issue and read the description properly. If you want to work on it, go to the comment section and comment that you want to work on it. One of our volunteers will assign you that issue.
+
+---
+
+### Working on the issue
+Now since you have your first issue assigned, we will be discussing some basic steps that need to be followed while working on the issue.
+
+*Steps 1 to 3 are for setup purposes only. You have to follow these steps only once on each project.*
+
+*Steps 4 to 9 should be repeated for every new contribution.*
+
+
+**Step 1: Forking** 
+
+Fork the main project repository by clicking on the **Fork** button at the top-right.
+Forking will create a copy of the repository in your Github account.
+![fork](https://user-images.githubusercontent.com/64194946/93016931-570b6500-f5e2-11ea-98fe-f226934091e4.jpg)
+
+
+**Step 2: Cloning**
+
+*For further steps, you need to have git installed. You can download git from [here](https://git-scm.com/downloads)*
+
+Now since you have git installed, run the following command to clone the repository: 
+```
+git clone https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME
```

Comment:
update PROJECT_REPOSITORY_NAME to `vms`  here and in commands below too as this doc is dedicated to vms only 
It's also easy to copy paste these commands then